src/libraries/System.Private.CoreLib/src/System/Runtime/GCFrameRegistration.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.Loader;
+
+namespace System.Runtime
+{
+#pragma warning disable 0414, IDE0044
+    internal unsafe struct GCFrameRegistration
+    {
+        private nuint m_reserved1;
+        private nuint m_reserved2;
+        private void** m_pObjRefs;
+        private uint m_numObjRefs;
+        private int m_MaybeInterior;
```

Comment:
```suggestion
        private nuint _reserved1;
        private nuint _reserved2;
        private void** _pObjRefs;
        private uint _numObjRefs;
        private int _maybeInterior;
```
This should be updated to follow libraries coding conventions now that it is under libraries.

Reply:
Done